x-pack/plugins/index_management/__jest__/client_integration/home/data_streams_tab.test.ts

```diff
@@ -370,21 +362,19 @@ describe('Data Streams tab', () => {
       testBed.component.update();
     });
 
-    describe('detail panel', () => {
```

Comment:
was it intentional to remove this describe() block?

Reply:
oh no! nice catch, I was trying to figure out why it was failing and must've forgotten to undo these changes